configure.py

```diff
@@ -57,6 +57,11 @@
 with open ('tools/icu/icu_versions.json') as f:
   icu_versions = json.load(f)
 
+sharable_builtins = {'cjs_module_lexer/lexer': 'deps/cjs-module-lexer/lexer.js',
+                     'cjs_module_lexer/dist/lexer': 'deps/cjs-module-lexer/dist/lexer.js',
+                     'undici/undici': 'deps/undici/undici.js'
```

Comment:
I assume this means that distros can now provide different versions of these dependencies. Are we okay with that?

Reply:
I see this as very similar to supporting external shared libraries and it is motivated by a similar use case. Since it does not affect what we ship I think it's up to the distros that use it to keep the versions the same or compatible in what they ship.

Reply:
I think we need to carefully weigh this. I see the motivation but not quite convinced yet.

Reply:
@jasnell can you outline your concerns in a bit more detail?  We have 2 distro's asking for the functionality in order to avoid having to hack in their own solutions?